yaks-rails/lib/yaks-rails.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails'
+require 'yaks-html'
+
+module Rails
+  module Yaks
+    def self.global_config
+      @global_config ||= ::Yaks.new
+    end
+
+    class << self
+      attr_writer :global_config
+    end
+
+    def self.configure(&block)
+      @global_config = ::Yaks.new(&block)
+    end
+
+    module ControllerAdditions
+        def yaks(object, opts = {})
+          runner = Yaks.global_config.runner(object, {env: env}.merge(opts))
+          puts runner.media_type
+          render body: runner.call, content_type: runner.media_type
+        end
+
+      def self.included(base)
+        base.helper_method :yaks if base.respond_to? :helper_method
+      end
+    end
+  end
+end
+
+if defined? ActionController::Base
+  ActionController::Base.class_eval do
+    include Rails::Yaks::ControllerAdditions
+  end
+end
```

Comment:
Actually, the Rails way to do it would be

```ruby
ActiveSupport.on_load(:action_controller) do
   include Rails::Yaks::ControllerAdditions
end
```
so it's only included, lazily, when actioncontroller is loaded